### ISSUE TITLE
winget installer does not include "Microsoft Visual C++ Redistributable"

### ISSUE THREAD

**Issue by pjking07** [volta-cli/volta#1856]
## Summary

After installing Volta from winget on a fresh Windows 11 instance, running `volta` yields a System Error.

> The code execution cannot proceed because VCRUNTIME140.dll was not found. Reinstalling the program may fix this problem.

| Name | Value |
|---|---|
| OS Name | Microsoft Windows 11 Pro |
| Version | 10.0.22631 Build 22631 |
| System Model | Parallels ARM Virtual Machine |
| System Type | ARM64-based PC |
| Chip | Apple M1 Pro |
| macOS | Sonoma 14.6.1 |

## Steps to reproduce
1. Enter a new Windows 11 VM in Parallels Desktop
2. Update winget to latest version via Microsoft Store
3. Install volta via `winget install Volta.Volta`
4. Open a new CMD terminal (PowerShell does not display the error)
5. Run `volta --version`
6. Observe System Error

<img width="516" alt="image" src="https://github.com/user-attachments/assets/fbb7418d-871e-4ea6-bf16-7095172fd28d">

## Workaround

Install `Microsoft Visual C++ Redistributable`

### x64

```sh
winget install Microsoft.VCRedist.2015+.x64
```

### Arm64

```sh
winget install Microsoft.VCRedist.2022.arm64
```

## Possible Solution

Update the winget Volta.Volta.installer.yaml to include `Microsoft Visual C++ Redistributable` as a package dependency.

https://github.com/microsoft/winget-pkgs/blob/2e2ba209c83c04fe947507d2bffbe6fa02ee996c/manifests/v/Volta/Volta/2.0.0/Volta.Volta.installer.yaml#L28-L33

```yaml
- Architecture: x64
  InstallerUrl: https://github.com/volta-cli/volta/releases/download/v2.0.0/volta-2.0.0-windows-x86_64.msi
  InstallerSha256: 11512614339C6563A567E17D12F9CEBBAB4E689158455A52BCCF19886466EB8D
  Dependencies:
    PackageDependencies:
    - PackageIdentifier: Microsoft.VCRedist.2015+.x64
- Architecture: arm64
  InstallerUrl: https://github.com/volta-cli/volta/releases/download/v2.0.0/volta-2.0.0-windows-arm64.msi
  InstallerSha256: 886DD84CAF7F4F107A2820D4A0F78BBCD6F81BBFCE5562DAF724E04F1F432FEA
  Dependencies:
    PackageDependencies:
    - PackageIdentifier: Microsoft.VCRedist.2022.arm64
```

**Comment by charlespierce** [comment]
Hi @pjking07, thanks for reporting and sorry about the inconvenience! I believe this is a duplicate of the most recent comments on #587. Just this week I merged a change to remove the need for that package entirely, so this should be fixed with the 2.0.1 release (ideally I'll be able to release that today, if not then it'll be over the weekend some time).

**Comment by pjking07** [comment]
Oh! I forgot about that old issue, and I didn't notice the updates while I was out. Thank you @charlespierce!

**Comment by charlespierce** [comment]
@pjking07 I just published Volta 2.0.1, which should solve this issue! The PR to add the installers into Winget is still processing, but once that is approved and merged it should be available. In the interim, the installers are available in the Releases page here on GitHub.
